Improve pre-upgrade documentation (#3855)
theforeman/foreman-documentation: guides/common/modules/proc_upgrading-smartproxy-server.adoc

[id="upgrading_{smart-proxy-context}_server_{context}"]
= Upgrading {SmartProxyServers}

ifdef::satellite[]
After you have upgraded your {ProjectServer} and synchronized the required repositories, you can start upgrading your {SmartProxyServers}.
endif::[]
You can upgrade from any minor version of {SmartProxyServer} {ProjectVersionPrevious} to {ProjectVersion}.

.Prerequisites
ifdef::katello,orcharhino,satellite[]

> * If you use content views to control updates to the base operating system of {SmartProxyServer}, or for {SmartProxyServer} repository, update those content views with new repositories, publish, and promote their updated versions.

Reviewer: ```suggestion
* If you use content views to control updates to the {SmartProxyServer} base operating system or repository, update those content views with new repositories, then publish and promote their updated versions.
```
